addons/project/static/src/js/tours/project.js

```diff
@@ -44,6 +44,7 @@ tour.register('project_tour', {
     run: "text Test",
 }, {
     trigger: ".o_kanban_project_tasks .o_column_quick_create .o_kanban_add",
+    content: Markup(_t('Let\'s create your first <b>stage</b>.')),
```

Comment:
Hello @odoo/rd-security ,

Can we have an override? 🙂 

We use Markup with static content in JS tour. 

Thanks in advance! 🙂 

Have a nice day!